doc/source/tutorial/tutorial_sessions.ipyml

```diff
@@ -470,16 +645,18 @@ cells:
     A more visual way is to use the [compare](../_generated/larray.compare.rst#larray.compare) function which will open the ``Editor``.
     
     ```python
-    compare(demography_session, demography_session_alternative, names=['baseline', 'lower_birth_rate'])
+    compare(s, s_alternative, names=['baseline', 'lower_birth_rate'])
     ```
     
     ![compare two sessions](../_static/compare_tutorial.png)
 
 
 - markdown: |
-    ## Session API
+    ## (Checked)Session API
```

Comment:
I know there are the parentheses, but I didn't realize at first this section was valid for (non checked) sessions too. I also **wonder** if the section is actually needed?

Reply:
Well, it's not no longer needed